tests/Doctrine/Tests/Common/Annotations/Annotation/TargetTest.php

```diff
@@ -35,13 +35,21 @@ public function testValidMixedTargets()
     {
         $target = new Target(array('value' => array('ALL')));
         self::assertEquals(Target::TARGET_ALL, $target->targets);
+        self::assertSame(['ALL'], $target->value);
+        self::assertSame('ALL', $target->literal);
+        self::assertInternalType('array', $target->value);
+        self::assertInternalType('string', $target->literal);
```

Comment:
Both type assertions are useless since that's already checked by using `assertSame` (as opposed to `assertEquals`).